ow_plexil/launch/ow_exec.launch

```diff
@@ -2,6 +2,11 @@
 
 <launch>
   <arg name="plan" default="None"/>
+  <arg name="fault_dependencies" default=""/>
+
+  <param name="fault_dependencies_file" type="string" value="$(arg fault_dependencies)"/>
+  <param name="fault_dependencies_path" type="string" value="$(find ow_plexil)/src/plans/"/>
+  <rosparam file="$(find ow_plexil)/config/fault_dependencies.yaml" />

```

Comment:
My question is now on this line -- how is this `rosparam` used, and why is it needed in addition to the `fault_dependencies` argument?

Reply:
The  ```fault_dependencies_file``` takes in the ```fault_dependencies``` argument (xml file name) and makes it a rosparam called ```fault_dependencies_file```. 

This is then used in conjunction with ```fault_dependencies_path``` to load the correct XML file in the FaultDependencies class.